src/Computer/Settings.cpp

```diff
@@ -28,6 +29,11 @@ PlacesOfInterestSettings::SetHome(const Waypoint &wp)
   home_waypoint = wp.id;
   home_location = wp.location;
   home_location_available = true;
+  if (wp.has_elevation) {
+    home_elevation = wp.elevation;
+    home_elevation_available = true;
+  } else
+    home_elevation_available = false;
 }
```

Comment:
This else isn't necessary.

Reply:
Are you sure? Without this, if the user changes home waypoints using the "Set as new home" button - changing from a waypoint WITH elevation data to one WITHOUT elevation data - wouldn't home_elevation_available wrongly remain TRUE (when it should be changed to FALSE)?